src/Commands/SetShooter.cpp

```diff
@@ -0,0 +1,35 @@
+#include "SetShooter.h"
+#include "../Subsystems/Shooter.h"
+std::shared_ptr<Shooter> shooter = Shooter::getInstance();
+SetShooter::SetShooter(double speed) : Command() {
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(Robot::chassis.get());
+	Requires(Shooter::getInstance().get());
+	mSpeed = speed;
+}
+
+// Called just before this Command runs the first time
+void SetShooter::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void SetShooter::Execute() {
+ shooter->SetShooter(mSpeed);
+}
+
+// Make this return true when this Command no longer needs to run execute()
+bool SetShooter::IsFinished() {
+	return false;
```

Comment:
Would you like me to just fix this command or make the command as a instant command?